oauth2/pom.xml

```diff
@@ -39,6 +39,12 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
```

Comment:
Context: https://github.com/okta/okta-spring-boot/pull/331#issuecomment-918758294

Reply:
A better import might be the Validation API:

```xml
<dependency>
    <groupId>javax.validation</groupId>
    <artifactId>validation-api</artifactId>
</dependency>
```

Then, change the import to:

```java
import javax.validation.constraints.NotNull;
```

WDYT @bdemers?

I tried this and all tests pass.

Reply:
Sounds good to me, updated PR.

Reply:
💯 The version will be managed automatically too.